google_auth_oauthlib/flow.py

```diff
@@ -200,9 +206,9 @@ def authorization_url(self, **kwargs):
                 :class:`Flow` instance to obtain the token, you will need to
                 specify the ``state`` when constructing the :class:`Flow`.
         """
+        kwargs.setdefault('access_type', 'offline')
```

Comment:
Probably worth updating the docstring to explain that `access_type=offline` will be added if not passed in?

Reply:
It's there.

Reply:
D'oh